weaviate/proto/v1/__init__.py

```diff
@@ -0,0 +1,4 @@
+import warnings
+
+warnings.filterwarnings("ignore", ".*obsolete", UserWarning, "google.protobuf.runtime_version")
```

Comment:
should we add a specific version here, so we do not hide real warnings?

Reply:
It only matches on the `google.protobuf.runtime_version` module so should not suppress other warnings emitted by our client. Or do you mean other real warnings from this module also?

Reply:
> Or do you mean other real warnings from their module also?

yes!

Reply:
I think the obsolete warning is valid in general, they just screwed it up in this specific case

Reply:
Looking at the warnings emitted by `google.protobuf.runtime_version.py`, the string `obsolete` is only present in the clause defined by `elif MINOR > gen_minor or PATCH > gen_patch:` so this seems like a well-targeted filter.

Is the concern that future warnings may be accidentally matched? I don't see a better way than what was recommended here in the original issue. What do you have in mind?

Reply:
> Is the concern that future warnings may be accidentally matched? I don't see a better way than what was recommended here in the original issue. What do you have in mind?

My concern is that version warnings are ignored in general. We should only ignore this specific case, eg add the version numbers into the matching

Reply:
Hmm, okay I see your point. But this problem will likely occur again when `protobuf` bumps its version number out-of-sync with `grpc-tools` in which case we'll end up with generated stubs in an old version used in the new runtime

I'm happy to handle that as and when it happens though!

Reply:
Have made the regex more restrictive, LMK what you think 😁

Reply:
thank you!

> But this problem will likely occur again when protobuf bumps its version number out-of-sync with grpc-tools in which case we'll end up with generated stubs in an old version used in the new runtime

yep, but I'd rather be careful here. Let's see how often it happens